### PR TITLE
Emagging a gravity generator will fuck everyone up.

### PR DESCRIPTION
**Pull request by silicons**
Wew.
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=9531&p=252103#p252103

### CODE REVIEW

**Comment by optimumtact** [comment]
client dir causes graphical issues because some sprites do not have the proper rotational sprites

File: code/modules/power/gravitygenerator.dm [hunk 1]
```diff
@@ -28,6 +30,44 @@ var/const/GRAV_NEEDS_WRENCH = 3
...
+
+/obj/machinery/gravity_generator/proc/fuck_my_shit_up()
+	chaos = TRUE
+	for(var/client/C in clients)
+		C.setDir(pick(cardinal))
+	for(var/mob/M in world)
+		M << "<span class='userdanger'>You feel sick as your feet leave the ground and you're flung across the room!</span>"
+		if(isliving(M))
+			M.throw_at(get_edge_target_turf(src, pick(cardinal)), 14, 3)
+			M.Weaken(5)
```
Comment (Crazylemon64) [inline code/modules/power/gravitygenerator.dm]: Bonus points for using matrix scaling to squish people flat

**Comment by kevinz000** [comment]
this was a joke pr that was for testmerging lmao.

**Comment by Armhulen** [comment]
:(

**Pull request closed without merging**